ci/etc/expected_install_directories

```diff
@@ -68,10 +71,13 @@
 ./include/google/cloud/logging
 ./include/google/cloud/logging/internal
 ./include/google/cloud/logging/mocks
+./include/google/cloud/orchestration
+./include/google/cloud/orchestration/airflow
+./include/google/cloud/orchestration/airflow/service
+./include/google/cloud/orchestration/airflow/service/v1
 ./include/google/cloud/orgpolicy
 ./include/google/cloud/orgpolicy/v2
 ./include/google/cloud/oslogin
-./include/google/cloud/oslogin/common
```

Comment:
Looks like I botched the rebase?

Reply:
No, I think regenerating the libraries at `HEAD` removes that entry.  I suspect that the scaffold step and the `generate-libraries-pr` produce different outputs or something like that.

Reply:
#7962